src/dev/build/tasks/nodejs/clean_node_builds_task.ts

```diff
@@ -15,12 +15,12 @@ export const CleanNodeBuilds: Task = {
     for (const platform of config.getTargetPlatforms()) {
       await deleteAll(
         [
-          build.resolvePathForPlatform(platform, '*/node/lib/node_modules'),
-          build.resolvePathForPlatform(platform, '*/node/bin/npm'),
-          build.resolvePathForPlatform(platform, '*/node/bin/npx'),
-          build.resolvePathForPlatform(platform, '*/node/bin/corepack'),
-          build.resolvePathForPlatform(platform, '*/node/CHANGELOG.md'),
-          build.resolvePathForPlatform(platform, '*/node/README.md'),
+          build.resolvePathForPlatform(platform, 'node/*/lib/node_modules'),
```

Comment:
correction, the cleanup paths were updated, just incorrectly :slightly_smiling_face: